src/ext.js

```diff
@@ -293,11 +288,6 @@ const definition = {
               defaultValue: true,
               show: false,
             },
-            cId: {
```

Comment:
Above here, autosort is still included, so it will be declared twice

Reply:
Deleted.